### ISSUE TITLE
Windows octo.exe does not perform delta transfers for uploads

### ISSUE THREAD

**Issue by mcasperson** [OctopusDeploy/Issues#5175]
When a package exists on the server, an `octo --push` should perform a delta package transfer. However the Windows Octo.exe fails with the error:

```
octo push --package package.1.0.0.nupkg --replace-existing
Octopus Deploy Command Line Tool, version 4.43.1
Detected automation environment: “NoneOrUnknown”
Handshaking with Octopus Server: https://example.octopus.app/
Handshake successful. Octopus version: 2018.9.9; API version: 3.0.0
Authenticated as: foobar user@example.com
Pushing package: package.1.0.0.nupkg…
Requesting signature for delta compression from the server for upload of a package with id ‘package’ and version ‘1.0.0’
Something went wrong while performing a delta transfer: Could not load file or assembly ‘Octodiff, Version=1.2.1.0, Culture=neutral, PublicKeyToken=null’ or one of its dependencies. The system cannot find the file specified.
Falling back to pushing the complete package to the server
Package transfer completed
```

It appears that the Octodiff.dll file is not being embedded into the exe properly. The DotNET Core packages, which include the Octodiff.dll file, work as expected.

**Comment by mcasperson** [comment]
Copying the Octodiff.dll file from one of the DotNET Core packages allows the standalone exe to perform a delta transfer.

**Comment by mcasperson** [comment]
This will be fixed in 4.43.2
